Formula/inko.rb

```diff
@@ -18,13 +18,16 @@ class Inko < Formula
   end
 
   depends_on "coreutils" => :build
+  depends_on "llvm@15" => :build
```

Comment:
any upstream issue tracking the llvm@16 build support?

Reply:
@chenrui333 There's no issue for it, and LLVM 16 probably won't be supported for some time. The LLVM library we're using doesn't support multiple LLVM versions at once, so you have to pick which one you want to support. Given LLVM 16 is pretty new, it's not nearly as widely available as LLVM 15. We'll probably update somewhere this year, but right now it's not a priority at all.

Reply:
sounds good 👍